code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -110,37 +110,22 @@
 /obj/item/weapon/storage/lockbox/emp_act(severity)
 	..()
 	if(!broken)
+		var/probab
 		switch(severity)
 			if(1)
-				if(prob(80))
-					locked = !locked
-					src.update_icon()
-					if(!locked)
-						for(var/atom/movable/A in src)
-							for(var/obj/O in src)
-							remove_from_storage(A, get_turf(src))
-						if(oneuse)
-							qdel(src)
+				probab = 80
 			if(2)
-				if(prob(50))
-					locked = !locked
-					src.update_icon()
-					if(!locked)
-						for(var/atom/movable/A in src)
-							for(var/obj/O in src)
-							remove_from_storage(A, get_turf(src))
-						if(oneuse)
-							qdel(src)
-			if(3)
-				if(prob(25))
-					locked = !locked
-					src.update_icon()
-					if(!locked)
-						for(var/atom/movable/A in src)
-							for(var/obj/O in src)
-							remove_from_storage(A, get_turf(src))
-						if(oneuse)
-							qdel(src)
+				probab = 50
+		if(prob(probab))
+			locked = !locked
+			src.update_icon()
+			if(!locked)
+				for(var/obj/O in src)
```

Comment:
This is probably fine in 99.9% cases but the rest of the code filters for `/atom/movable` as far as I can tell, rather than `/obj`, so change it to that, I guess.

Reply:
Change the cases of /atom/movable to /obj, or the other way around? Not sure what you mean by "that".

Reply:
Change this one to `/atom/movable`